tests/src/test/java/alluxio/master/file/FileSystemMasterIntegrationTest.java

```diff
@@ -846,6 +848,8 @@ public void unavailableUfsRecursiveCreate() throws Exception {
         .thenReturn(new UfsDirectoryStatus("test", "owner", "group", (short) 511));
     Mockito.when(mockUfs.mkdirs(Matchers.eq(ufsBase + "/dir1"), Matchers.any()))
         .thenThrow(new IOException("ufs unavailable"));
+    Mockito.when(mockUfs.getStatus(Matchers.any()))
```

Comment:
we might want to mock different file statuses in the future - better to match the exact string that we want to map to `new UfsDirectoryStatus("test", "test", "test", (short) 0755)`

Reply:
Done.